apps/appcenter-post-clone.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 echo "Executing post clone script in `pwd`"
 echo "dimension.dependency=$DIMENSION_DEPENDENCY" >> $APPCENTER_SOURCE_DIRECTORY/local.properties
+../scripts/set-ndk-version.sh
```

Comment:
BTW why that script is in `scripts` folder but changes the app only?

Reply:
I think it's more convenient to store them all in one folder